app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/DeploymentGraphService.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend.whatsrunningwhere
+
+import uk.gov.hmrc.cataloguefrontend.model.Environment
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.UpstreamErrorResponse.Upstream4xxResponse
+
+import java.time._
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class DeploymentGraphService @Inject() (releasesConnector: ReleasesConnector)(implicit ec: ExecutionContext) {
+
+  // search 60 days either side of the time-range, this allows us to find deployments that intersect the time-range
```

Comment:
60 or 90 days?

Reply:
90. though it doesnt matter too much, there's isnt much difference performance-wise